README.md

```diff
@@ -101,10 +134,6 @@ Console.WriteLine("prediction: " + prediction.Prediction);
 ```
 A cookbook that shows how to use these APIs for a variety of existing and new scenarios can be found [here](docs/code/MlNetCookBook.md).
 
-## Samples
-
-We have a [repo of samples](https://github.com/dotnet/machinelearning-samples) that you can look at.
-
 ## License
 
 ML.NET is licensed under the [MIT license](LICENSE).
```

Comment:
should we explicitly mentioned that it is free to use commercially?

Reply:
Done! 👍